features-json/heif.json

```diff
@@ -555,13 +555,13 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"While supported natively since macOS 10.13 High Sierra and iOS 11, the file format does not appear to be supported in Safari."
+    "1":"Was supported natively in macOS/iOS, but was not supported in Safari."
```

Comment:
Safari 17 added support for HEIC/HEIF images: https://developer.apple.com/documentation/safari-release-notes/safari-17-release-notes#Images

Reply:
Correct, that was added in #6738. That is why I have changed the sentence from present tense to past tense.